ext/standard/head.c

```diff
@@ -148,7 +148,12 @@ PHPAPI int php_setcookie(zend_string *name, zend_string *value, time_t expires,
 			strlcat(cookie, ZSTR_VAL(dt), len + 100);
 			zend_string_free(dt);
 
-			snprintf(tsdelta, sizeof(tsdelta), ZEND_LONG_FMT, (zend_long) difftime(expires, time(NULL)));
+			int diff;
+			diff = difftime(expires, time(NULL));
+			if (diff < 0) {
+				diff = 0;
```

Comment:
hm you are right... the docs state that a value in the past should be used when the cookies needs to be deleted. therefore a warning/notice doesnt fit in here.

leave it as is ;)